docs/code-of-conduct.md

```diff
@@ -0,0 +1,89 @@
+include:
+
+* Using welcoming and inclusive language
+* Being friendly and patient
+* Being respectful of differing viewpoints and experiences, and trying to
+  understand other people's points of view
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+* Assuming good faith on the part of other community members
+* Respecting the time and attention of others, in particular by doing research
+  in FAQs and with search engines before asking questions of maintainers
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or
+  advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Criticism of contributions which is not constructive, which occurs outside
+  project channels or which is repetitive
+* Publishing others' private information, such as a physical or electronic
+  address without explicit permission
+
+We will exclude you from interaction if you insult, demean or harass anyone.
+That is not welcome behavior.
+
+## Responsibilities of Project Maintainers
+
+Project maintainers are responsible for clarifying the standards of acceptable
+behavior and are expected to take appropriate and fair corrective action in
+response to any instances of unacceptable behavior.
+
+Project maintainers have the right and responsibility to remove, edit, or reject
+comments, commits, code, wiki edits, issues, and other contributions that are
+not aligned to this Code of Conduct, or to ban temporarily or permanently any
+contributor for other behaviors that they deem inappropriate, threatening,
+offensive, or harmful.
+
+## Scope
+
+This Code of Conduct applies both within project spaces and in public spaces
+when an individual is representing the project or its community. Examples of
+project spaces include mailing lists, chat.zulip.org, GitHub and in code.
+Examples of representing a project or community include using an official
+project e-mail address, posting via an official social media account, or acting
+as an appointed representative at an online or offline event.
+
+## Enforcement
+
+If you feel you have been or are being harassed or made uncomfortable by a
+community member, please contact zulip-core@googlegroups.com immediately.
+Whether you're a regular contributor or a newcomer, we care about making this
+community a safe place for you and we've got your back.
+
+We encourage anyone who observes abusive, harassing, or otherwise unacceptable
+behavior to report that by emailing zulip-core@googlegroups.com. All reports
+will be reviewed and investigated and will result in a response that is deemed
+necessary and appropriate to the circumstances by the project maintainers. The
+project team will keep confidential the details of the reporter.
```

Comment:
I would re-word or omit this completely because keeping something completely confidential may not be possible.